stable/kuberhealthy/values.yaml

```diff
@@ -21,6 +21,22 @@ resources:
     cpu: 400m
     memory: 200Mi
 
+# Disable certain checks by setting enabled to false
+componentStatusCheck:
```

Comment:
This isn't bad, but what about a structure like this?

```
checksEnabled:
  daemonSetCheck: true
  dnsStatusCheck: true
  componentStatusCheck: true
  podRestartsCheck: true
  podStatusCheck: true
```

Reply:
I suppose your way would be better if we wanted to later on include more options for each check, so I think the way you have it is probably fine - just giving random feedback.

Reply:
Hmm, if we wanted to add more options for each check would we be able to do:
```
anotherOption:
  daemonSetCheck: value
  dnsStatusCheck: value
  componentStatusCheck: value
  podRestartsCheck: value
  podStatusCheck: value
```
??
I guess I'm wondering which structure is better/more appropriate?

Reply:
I think the way you did it is fine!  In the future, you would be able to do stuff like this:

```
daemonSetCheck:
  enabled: true
  interval: 30m
  image: image.url.here
```

Forget I said anything!